internal/database/security_event_logs.go

```diff
@@ -58,3 +60,17 @@ func (s *SecurityEventLogStore) Insert(ctx context.Context, e *SecurityEvent) er
 	}
 	return nil
 }
+
+// LogAuthEvent will log an authentication or an authorization event. Examples are:
+// SignOutAttempted, SignOutScceeded, SignOutFailed.
+func (s *SecurityEventLogStore) LogAuthEvent(ctx context.Context, e *SecurityEvent) {
```

Comment:
```suggestion
func (s *SecurityEventLogStore) LogSecurityEvent(ctx context.Context, e *SecurityEvent) {
```

Doesn't look like this is specific to Auth

Reply:
Based on the usage `database.SecurityEventLogs(db).LogAuthEvent(ctx, event)`, I think this could be as simple as follows:
```suggestion
func (s *SecurityEventLogStore) LogEvent(ctx context.Context, e *SecurityEvent) {
```